### ISSUE TITLE
Replace System.Net.Mail.SmtpClient

### ISSUE THREAD

**Issue by kamronbatman** [modernuo/ModernUO#19]
Replace email log code to fix the following warnings:
```
`System.Net.Mail.SmtpClient' is obsolete: `SmtpClient and its network of types are poorly designed, we strongly recommend you use https://github.com/jstedfast/MailKit and https://github.com/jstedfast/MimeKit instead'
```

**Comment by kamronbatman** [comment]
This should be entirely removed and replaced with a separate module.

**Comment by 3HMonkey** [comment]
Currently working on this using MailKit...
![grafik](https://user-images.githubusercontent.com/4610892/71664866-fb9e8d00-2d5a-11ea-99bf-add581da8bb4.png)

**Comment by kamronbatman** [comment]
Awesome thank you!

**Comment by 3HMonkey** [comment]
https://github.com/modernuo/ModernUO/pull/69

**Comment by kamronbatman** [comment]
Closed in #69
